lib/_tls_common.js

```diff
@@ -148,10 +150,31 @@ exports.createSecureContext = function createSecureContext(options) {
     }
   }
 
-  if (options.ciphers)
-    c.context.setCiphers(options.ciphers);
-  else
-    c.context.setCiphers(tls.DEFAULT_CIPHERS);
+  if (options.ciphers && typeof options.ciphers !== 'string') {
+    throw new ERR_INVALID_ARG_TYPE(
+      'options.ciphers', 'string', options.ciphers);
+  }
+
+  // Work around an OpenSSL API quirk. cipherList is for TLSv1.2 and below,
+  // cipherSuites is for TLSv1.3 (and presumably any later versions). TLSv1.3
+  // cipher suites all have a standard name format beginning with TLS_, so split
+  // the ciphers and pass them to the appropriate API.
+  const ciphers = (options.ciphers || tls.DEFAULT_CIPHERS).split(':');
+  const cipherList = ciphers.filter((_) => !_.match(/^TLS_/)).join(':');
+  const cipherSuites = ciphers.filter((_) => _.match(/^TLS_/)).join(':');
```

Comment:
This feels like a complete hack but I'm not sure there's a better way unless we expose two different options for setting the cipher suites.

Reply:
I couldn't think of a better way, and I think two options would be awful. This code may be odd, but its only a couple lines long, and its very robust. We could go the BoringSSL way, and not even allow the TLS1.3 suites to be configurable, but that seems like not such a great idea, either. IoT devs would not be happy.